lib/Controllers/list.js

```diff
@@ -61,14 +61,23 @@ List.prototype.fetch = function(req, res, context) {
     var searchOperator = this.resource.search.operator || '$like';
     var searchAttributes =
       this.resource.search.attributes || Object.keys(model.rawAttributes);
+    var stringOperators = {
```

Comment:
why not use a regex here instead? relevant benchmark: http://jsperf.com/key-in-vs-match